sssom/parsers.py

```diff
@@ -37,6 +37,8 @@ def from_tsv(
     """
     if validators.url(file_path) or os.path.exists(file_path):
         df = read_pandas(file_path)
+        if 'confidence' in df.columns:
```

Comment:
why is this change here?

Reply:
Apparently `msdf.df` from some test tsvs do not have a 'confidence' column. If this makes no sense I can dig deeper.

Reply:
I dont understand what this has to do with the input argument ticket :), that's all.

Reply:
ohhh ... so basically when I switched branches, for some reason changes from #100  also got included in spite of those changes being on a different branch. Have you seen experienced something like this before? Am I doing something wrong to cause this?

Reply:
Probably just an accident being on the wrong branch when running git checkout -b..

Reply:
Possible ...sorry about that.